inc/wc-calypso-bridge-products.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Adds additional meta to products for the Store on .com experience.
+ * See also wc-calypso-bridge-jetpack-sync.php
+ *
+ * @since 0.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+function wc_calypso_bridge_products_get_context() {
+	if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
+		$referrer = isset( $_SERVER['HTTP_REFERER'] ) ? $_SERVER['HTTP_REFERER'] : '';
+		$known_referrers = array(
+			'page=product_importer' => 'ajax-product-importer',
+			'/post-new.php'         => 'ajax-post-new',
+		);
+
+		foreach ( $known_referrers as $known_referrer => $context ) {
+			if ( false !== strpos( $referrer, $known_referrer ) ) {
+				return $context;
+			}
+		}
+
+		return 'ajax-unknown';
+	}
+
+	$request_uri = isset( $_SERVER['REQUEST_URI'] ) ? $_SERVER['REQUEST_URI'] : '';
+	$known_fragments = array(
+		'/wp-json/wc/v2'          => 'rest-api-v2',
```

Comment:
We should probably support v1 as well. There are also legacy endpoint versions before that, though I think it's more likely v1 is still in use.